packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -615,6 +615,59 @@ function getRank(
   return getRankOrder(memberGroups, orderConfig);
 }
 
+/**
+ * If there are more than 1 consecutive same ranked member in 1 class, those members are put in an array & an array of these members' array is returned as a result. If, for example, the memberSet parameter looks like the below example - (contd. after the example tag)
+ * @example
+ * ```
+ * interface Foo {
+      [a: string]: number;
+ 
+      a: x;
+      B: x;
+      c: x;
+ 
+      c(): void;
+      B(): void;
+      a(): void;
+ 
+      (): Baz;
+ 
+      new (): Bar;
+    }
+ * ```
+  The resulting array will look like this - [[a, B, c], [c, B, a]].
```

Comment:
[Praise] This example & surrounding text was super helpful, thank you very much for writing it! 🙌